lib/network_utils.pm

```diff
@@ -51,16 +51,19 @@ Returns if can ping worker host gateway
 =cut
 sub can_upload_logs {
     my ($gw) = @_;
-    $gw ||= $default_gw;
-    return script_run('ping -c 1 ' . $default_gw);
+    $gw ||= testapi::host_ip();
```

Comment:
in this case drop `$default_gw`  global variable and just replace only actual usage in line 32 with value itself

Reply:
Also though so. So, it's done. Please take a look, I've updated verification runs.